javaparser-core/src/main/java/com/github/javaparser/utils/VisitorSet.java

```diff
@@ -149,12 +149,9 @@ public <T> T[] toArray(T[] arr) {
 
     @Override
     public String toString() {
-        StringBuilder sb = new StringBuilder("[");
-        if (size() == 0) return sb.append("]").toString();
-        for (EqualsHashcodeOverridingFacade facade : innerSet) {
-            sb.append(facade.overridden.toString() + ",");
-        }
-        return sb.replace(sb.length() - 2, sb.length(), "]").toString();
+        return innerSet.stream()
+                .map(facade -> facade.overridden.toString())
+                .collect(Collectors.joining(", ", "[", "]"));
```

Comment:
I can't accept this PR, even though it undoubtedly improves the code, because you're changing the format of the character string returned by this method, which may have an impact on other users. Please only correct the issue you have identified.

Reply:
Sure, I have solved it and the format is the same as before.